content/03.results.md

```diff
@@ -114,69 +129,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
+We observed that 52/54 (96.3%) of tumors only have a mutation in either gene (OR = 0.188, 95% CI: 0.04 - 0.94, p = 4.13e-2, q = 5.87e-2 **TODO THIS IS NOT SIGNIFICANT??**), recapitulating a previously observed trend that _TP53_ and _PPM1D_ mutations tend to be mutually exclusive in HGGs [@doi:10.1038/ng.2938].
+
+We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medulloblastomas, had the most unstable genomes (**Figure {@fig:S3}A**).
+By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
+Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
+The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
+We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas and in 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
+We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors. **TODO: 15% of EACH of these groups, or TOTAL 15%?**
+
+We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures from the RegSig database [@doi:10.1038/s43018-020-0027-5] across samples (**Figure {@fig:Fig3}E** and **Figure {@fig:S4}A**).
+Stage 0 and/or 1 tumors characterized by low TMBs (**Figure {@fig:S2}G**) such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas were expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
+Signature N6 is a CNS-specific signature which we observed nearly universally across samples.
+Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observed Signature 18 as the most common signature in medulloblastoma tumors.
+Signatures 3, 8, 18, and MMR2 were prevalent in HGGs, including DMGs.
+Finally, we found that the exposure to Signature 1 was higher at diagnosis (pre-treatment) and was almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy; **Figure {@fig:S4}B**).
+This trend may have resulted from therapy-induced mutations that produced additional signatures (e.g., temozolomide treatment drives Signature 11 **TODO: this is not a strong claim; the refsig website says this finding is not reproducible.**), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.
```

Comment:
Note I'm deleting the temozolomide parenthetical. This was the reference which explains there is no strong evidence for this association - https://signal.mutationalsignatures.com/explore/referenceCancerSignature/40?studyId=1

> This signature was previously associated with temozolomide exposure, having been first identified in patients that had temozolomide during treatment of their intracranial tumours. However, the signature has not been recapitulated in experimental model systems. In spite of multiple independent experiments with temozolomide exposure, the signature was consistently different (see Temozolomide 200 µM (A) Substitution Signature). Intriguingly, the signature of treatment with 1,2-DMH was found to be very similar to this signature (see 1,2-Dimethylhydrazine 11.6 mM Rat S9 Rat S9 Substitution Signature).

Reply:
I suggest we instead write, "suggested to drive". I do understand that _one_ paper shows that this signature isn't TMZ, but model systems are not perfect. The paper says they used `human-induced pluripotent stem cells`, but perhaps they are not recapitulating brain cell diversity and the mutational signatures are not recapitulated perfectly. I looked at [this paper](https://pubmed.ncbi.nlm.nih.gov/25088490/v) to learn a bit more about DMH (I can send you if you would like). It states that 
>A methylating agent that preferentially targets the colon is 1,2-dimethylhydrazine (DMH). 

So if the COSMIC Sig 11 is due to DMH, which targets the colon, it wouldn't make sense for that signature to be in brain tumor samples at all. I am just not wholly convinced that Sig 11, which is observed in the samples we know were treated with TMZ, is not associated with TMZ exposure. 

I guess I am a bit weary of one pub challenging _years_ of "previously hypothesized truth" (not saying it isn't possible or isn't real), but it might be contextual (in this model, it produces a signature like this) or more complex if perhaps signature 11 is due to TMZ+radiation for example, but not TMZ or radiation alone, and they didn't test combinations. So I would feel more comfortable acknowledging this paper exists and says one thing contrary to all other known literature, but still acknowledging that this is suggested as a mechanism driving Sig 11.